podium-gateway/src/main/webapp/app/shared/request/request-review-panel/request-review-panel.component.html

```diff
@@ -0,0 +1,36 @@
+<h3 jhiTranslate="request.review.summaryTitle">Review Summary</h3>
+<table class="table">
+    <thead>
+    <tr>
+        <th jhiTranslate="request.review.reviewerName">Name</th>
+        <th jhiTranslate="request.review.date">Date</th>
+        <th jhiTranslate="request.review.result">Result</th>
+        <th jhiTranslate="request.review.comment">Comment</th>
+    </tr>
+    </thead>
+    <tbody *ngIf="lastReviewFeedback">
+    <tr *ngFor="let feedback of lastReviewFeedback">
+        <th scope="row">{{feedback.reviewer.firstName}} {{feedback.reviewer.lastName}}</th>
+        <td>{{feedback.date | date:'short'}}</td>
+        <td><span class="tag" [ngClass]="{
+                'tag-success': feedback.advice == 'Approved',
```

Comment:
Nice to have the comparison logic in the component. A function like isAdvised(String)